docs/dashboards/README.md

```diff
@@ -0,0 +1,237 @@
+See the diagram below for an example architecture.
+The `sidecar-collector` and `gateway-collectors` are used to send application telemetry to Dynatrace (e.g. traces, metrics, logs that the application produces).
+Both of these collectors send only their selfmonitoring data (or internal telemetry) to the `selfmon-collector`.
+The `selfmon-collector` is responsible for collecting, transforming and forwarding the internal telemetry for all collectors (including the `selfmon-collector` itself).
+Only monitoring data about the collector passes through it; no monitoring data from non-collector applicaitons is sent via the `selfmon-collector`.
+
+```mermaid
+flowchart LR
+    subgraph legend[Legend]
+        direction LR
+        leg1[ ]:::hide-- collector selfmonitoring data -->leg2[ ]:::hide
+        leg3[ ]:::hide-. application telemetry data .-> leg4[ ]:::hide
+        classDef hide height:0px
+    end
+
+    selfmon-collector:::collector
+
+    selfmon-collector-->Dynatrace
+    selfmon-collector-->selfmon-collector
+
+    sidecar-collector:::collector-->selfmon-collector
+
+    subgraph sg-sidecar[Application with a sidecar collector]
+    application -.-> sidecar-collector
+    end
+
+    gateway-collector:::collector-->selfmon-collector
+
+    sidecar-collector-.->Dynatrace
+    gateway-collector-.->Dynatrace
+
+    otherApps[Applications without sidecar collector]-.->gateway-collector
+    
+    style application fill:#adc9ff,stroke:#1966FF
+    classDef collector fill:#9afee0,stroke:#02D394
+    style sg-sidecar fill:#C2C2C2,stroke:#707070
+    style otherApps fill:#C2C2C2,stroke:#707070
+    style Dynatrace fill:#dcc2ff,stroke:#7F1AFF
+    style legend fill:#f2f2f2,stroke:#C2C2C2
+```
+
+Below, you can see a configuration example for a selfmonitoring collector.
+
+```yaml
+# receive selfmonitoring data via gRPC from OTel collector instances.
+receivers:
+  otlp/selfmon:
+    protocols:
+      grpc: 
+        endpoint: 0.0.0.0:4317
+
+processors:
+  # transform cumulative values to deltas. 
+  cumulativetodelta/selfmon: {}
+  # (kubernetes only) retrieves kubernetes attributes for other collectors sending to this collector. See Kubernetes prerequisites below.
+  k8sattributes/selfmon: {}
+
+  # prepend 'sfm.otelcol' to all selfmon metrics - the charts in the {dashboard_name}.json file expects this prefix.
+  transform/selfmon:
+    error_mode: ignore
+    metric_statements:
+      - context: metric
+        statements:
+          - set(name, Concat(["sfm.otelcol", name], "."))
+
+exporters:
+  # Inject DT_ENDPOINT and DT_API_TOKEN as environment variables. This should be the environment where the selfmonitoring data will go.
+  # See <https://docs.dynatrace.com/docs/shortlink/otel-getstarted-otlpexport> for instructions on which endpoint and token scope to use.
+  otlphttp/selfmon:
+    endpoint: "${DT_ENDPOINT}/api/v2/otlp"
```

Comment:
The endpoint and api token env variables should follow the same format as in our other use cases, for example https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/collector/use-cases/batch